client/landing/stepper/declarative-flow/internals/steps-repository/importer/index.tsx

```diff
@@ -87,6 +83,10 @@ export function withImporterWrapper( Importer: ImporterCompType ) {
 		const fromSiteData = useSelector( getUrlData );
 		const stepNavigator = useStepNavigator( flow, navigation, siteId, siteSlug, fromSite );
 		const currentPath = window.location.pathname + window.location.search;
+		const isRequestingAllSites = useSelector( isRequestingSites );
+		const isRequestingCurrentSite = useSelector( ( state ) =>
+			siteId ? isRequestingSite( state, siteId ) : true
+		);
```

Comment:
I have concerns about this being true when we don't have a `siteId` defined. Could you share more about why you set it up this way?

Reply:
It makes our UI show the loading state while we don't have any further decisions. 
But I don't have strong opinions about.

Reply:
Updated here [3547f17](https://github.com/Automattic/wp-calypso/pull/102277/commits/3547f172ccb3a78b66fd206155be46e7c0e0d202)